modules/apps/foundation/frontend-theme/responsive-boat-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,278 @@
+/* These inject tags are used for dynamically creating imports for themelet styles, you can place them where ever you like in this file. */
+
+/* inject:imports */
+/* endinject */
+
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+
+
+/********** VARIABLE SECTION **********/
```

Comment:
CSS comments should use a standard format.

Ex.
```
/* ---------- Variables ---------- */
```

Reply:
We do not need a comment around the variable since we can already tell that they are SASS variables.

Reply:
Line break needed below CSS comment.